### ISSUE TITLE
Minimum charging power of 3,7kW

### ISSUE THREAD

**Issue by MisterAtoZ** [thomase1234/esphome-fake-xemex-csmb#17]
Hi @thomase1234,

First and foremost thank you for sharing your code! I would like to continue your work and apply it to my wall box.

First, some background about my installation. I have the 3 phase 22kW Newmotion advanced installed with the DPM module. In the installation are solar panels and a home battery which I would like to include in the setup. Because Shell/Newmotion doesn’t offer any “smart” dynamic load balancing that you can configure yourself, I am very interested in your project to create it myself.

With the help of your code I was able to flash my esp32 and read/set the values in home assistant. I haven’t added any automations (yet). As I would like to test the system statically first. I also need to gain some more information about the wall box like the maximum current it is set to.

When trying this, I encountered a different issue that I didn’t really expect. It seems to me that the wall box has a minimum charging speed/power? When playing with the values from CT1, CT2, and CT3. I noticed that I can’t get the charging power under 3,7kW. I can increase the maximum current until the charging power is 3,7kW, but once I increase the current further, the wall box will just stop charging completely… For me, it looks like there is a minimum charging power of 3,7kW? Has anyone noticed the same?
To get the wall box charging again, I have to decrease the current drastically. It does not start charging again with the same current to get 3,7kW. I need to decrease the current a lot (a couple of amps per phase max) to get it to charge again. Once started, I can decrease the charging power to 3,7Kw again.

There is not much documentation about the wall box and help from Shell is not to be expected as they keep everything behind closed doors. Does anyone have the same experience? Or are there some other things that I can check?

Thanks for you help!

**Comment by virility0** [comment]
Hi!

Have you entered the front end directly on the wall box? 

Http://localip:12800/user/user.html

There you should see the needed current and the max current, if a car is connected.

What is very interesting for me is that you can decrease your wallbox to 3,7kw with 3 phases. 

3,2kw would be 8A on all 3 phases. 8A is the absolut Minimum.

On 3 phases mine does not go below 14A :-( don't know why..

**Comment by virility0** [comment]
Yes, 1 phase is also limited to 8A, which means about 1,8kW . 

Unfortunately there is no option to switch between 3 Phase and 1 Phase charging without changing the cable  -_-

**Comment by MisterAtoZ** [comment]
I do not really have the possibility to change cables because I have the cable attached to the wall box. So it would be a lot of work to change the cable each time.

I was able to connect the Ethernet cable and see the page you mentioned. Below are the results. The currents I send to the wall box via the esp are: 
```
ct1: 16,0A
ct2: 16,0A
ct3: 15,7A
``` 
This seems to be the highest I can go. Once I increase ct3 to 15,8A, it will stop. When checking the page of the wall box, I can see the values are as follows when the currents from above are set:
```
maxLimit: 32 Amp 3 phase
Max: 20 Amp 3 phase
Max Plug: /
Limited now: 8.050000190734863 Amp 3 phase
Charging rate: Charging rate: 5.54kw 3 phase [8.03A 0.97pf ] 
Charging need: Need: -0kw no [-1A ] 
```
The charging rate is weird here as the car is charging at 3,6-3,7kW. Not sure what the charging need is supposed to be.

Another funny thing I noticed is that when I set `ct1` and `ct2` to 1A, the limit stays the same, it will not charge with any more power. As soon as I touch `ct3`, the amount of power available changes. This makes me wonder if my car is only capable of charging with 1 phase. I do have a hybrid with a maximum charge power of 7,4kW.
In any case, your theory about it not going under 8A seems to be correct. Still a pity it's not possible to go under it.
